ci/build.yml

```diff
@@ -150,15 +150,10 @@ jobs:
         trigger_sha: '$(trigger_sha)'
     - template: report-end.yml
 
-- template: patch_bazel_windows/compile.yml
```

Comment:
why drop that?

Reply:
IT's not actually applying any patch anymore. The patch was removed in https://github.com/digital-asset/daml/pull/8798.

I'm not going to mention how long I spent trying to get the build to work on 2022 before noticing that, as that would be a bit embarrassing.

Reply:
hm fair enough, I was aware of that but so far we somewhat intentionally kept the infrasturcture for handling patches. don’t feel strongly about keeping it though.